wiki/Tournaments/IGTS/2018/en.md

```diff
@@ -57,12 +57,20 @@ The Intermediate Global Taiko Showdown 2018 is run by various osu!taiko communit
 
 ## Match Schedule: Finals - Week 1
 
-### Saturday, 8 September 2018
+### Saturday, 15 September 2018
 
 | Match ID | Team A |  |  | Team B | Match Time (UTC) |
 | :-: | --: | :-: | :-: | :-- | :-: |
-| #1 | Malaysia | ![][flag_MY] | ![][flag_NL] | Netherlands | **10:00** |
-| #2 | United States | ![][flag_US] | ![][flag_DE] | Germany | **20:00** |
+| #1 | Malaysia | ![][flag_MY] | ![][flag_US] | United States | **03:00** |
+
+### Sunday, 16 September 2018
+
+| Match ID | Team A |  |  | Team B | Match Time (UTC) |  |
+| :-: | --: | :-: | :-: | :-- | :-: | :-: |
+| #2a | Netherlands ![][flag_NL] | ![][flag_MY] Malaysia | **13:00** | ¹ |
```

Comment:
needs extra columns to fix table
`| **13:00** | ¹ |` should be `| **13:00** |  |  | ¹ |`